x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/components/phases/shared_fields/min_age_field/min_age_field.tsx

```diff
@@ -110,6 +119,26 @@ export const MinAgeField: FunctionComponent<Props> = ({ phase }): React.ReactEle
                         const { isInvalid: isUnitFieldInvalid } = getFieldValidityAndErrorMessage(
                           unitField
                         );
+                        const selectAppendValue: Array<string | React.ReactElement> = [
+                          i18n.translate(
+                            'xpack.indexLifecycleMgmt.editPolicy.minimumAge.minimumAgeFieldSuffixLabel',
+                            { defaultMessage: 'old' }
+                          ),
+                        ].concat(
+                          isUsingRollover
+                            ? [
+                                <>
+                                  {/* This element is rendered for testing purposes only */}
+                                  <div data-test-subj={`${phase}-rolloverMinAgeInputIconTip`} />
+                                  <EuiIconTip
+                                    type="iInCircle"
+                                    aria-label={i18nTexts.rolloverToolTipDescription}
+                                    content={i18nTexts.rolloverToolTipDescription}
+                                  />
+                                </>,
+                              ]
+                            : ([] as any[])
+                        );
```

Comment:
Hi @jloleysens , thanks for adding this tooltip! Could you please consider making this part of the code a little bit more straightforward? Something like: 
```
const oldLabel = ...
const tooltip = ...
const appendValue = isUsingRollover ? [oldLabel, tooltip] : [oldLabel]

```
We would also avoid another `any` ;)

Reply:
That's a good point!